qutebrowser.json

```diff
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://www.qutebrowser.org/",
+    "license": "https://github.com/qutebrowser/qutebrowser/blob/master/LICENSE",
+    "version": "1.1.1",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/qutebrowser/qutebrowser/releases/download/v1.1.1/qutebrowser-1.1.1-windows-standalone-amd64.zip",
+            "hash": "9c9693e7b7c28266b4edcf6de90ae15af3bde4aed4359149a25f6942f3b186a4",
+            "extract_dir": "qutebrowser-1.1.1-x64"
+        },
+        "32bit": {
+            "url": "https://github.com/qutebrowser/qutebrowser/releases/download/v1.1.1/qutebrowser-1.1.1-windows-standalone-win32.zip",
+            "hash": "27a0fae3706c8c4b5381f43b91e57bd3eb0c9f4a6594b97d84c22e01211fa44c",
+            "extract_dir": "qutebrowser-1.1.1-x64"
```

Comment:
Not quite there yet 😄
`extract_dir` has to be added to `autoupdate` as well.

Like so:
```
"url": "...",
"extract_dir": "qutebrowser-$version-x64"
```

Reply:
Adding persistence to the `config` directory would be a good idea too. 😁 
```json
"persist": ["config"]
```

Reply:
These little typos are killing me! 🙄